### PR TITLE
0.13: Exposure

### PR DESCRIPTION
**Pull request by cart**
Part of #891

### CODE REVIEW

File: content/news/2024-02-03-bevy-0.13/index.md [hunk 1]
```diff
@@ -834,11 +834,34 @@ commands.spawn((
...
 ```
 
-## Exposure Settings
+## Camera Exposure
 
-<div class="release-feature-authors">authors: @TODO</div>
+<div class="release-feature-authors">authors: @superdump (Rob Swain), @JMS55, @cart</div>
 
-TODO.
+Cameras in the "real world" control how bright an image is by adjusting "exposure settings":
```
Comment (alice-i-cecile) [inline content/news/2024-02-03-bevy-0.13/index.md:841]:
```suggestion
Cameras in the "real world" control how bright an image is by adjusting several "exposure settings":
```

Comment (alice-i-cecile) [inline content/news/2024-02-03-bevy-0.13/index.md:841]: I'd like to see a bit of a shout out to the whole idea of PBR: why are we trying to emulate real cameras at all?

Comment (cart) [reply content/news/2024-02-03-bevy-0.13/index.md:841]: Isn't "several" implied by the plural "settings" and the list that follows? Is this a grammar rule I'm not aware of?

Comment (TrialDragon) [reply content/news/2024-02-03-bevy-0.13/index.md:841]: "Settings" is the plural, yes; however, I think the usage here was more in line with "separate and distinct". That exposure is gotten by adjusting different settings in conjunction? Not sure if it's necessary here tho.

Comment (alice-i-cecile) [reply content/news/2024-02-03-bevy-0.13/index.md:841]: Part of the broader confusion here for those unfamiliar with photography is "why do we need multiple"? "Can't we just set a brightness level?"

This was intended to push this section in that direction :) I should have been more clear, sorry.

Comment (TrialDragon) [inline content/news/2024-02-03-bevy-0.13/index.md:841]:
```suggestion
In the "real world", the brightness of an image captured by a camera is determined by it's exposure: the amount of light that reaches the camera's sensor or film. This is controlled by several mechanics of the camera:
```

File: content/news/2024-02-03-bevy-0.13/index.md [hunk 2]
```diff
@@ -834,11 +834,34 @@ commands.spawn((
...
+
+* **F-Stops**: The size of the aperture, which opens and closes to control how much light is allowed into the camera by physically blocking off lights from specific angles.
```
Comment (TrialDragon) [inline content/news/2024-02-03-bevy-0.13/index.md:843]:
```suggestion
* **Aperture**: Measured in `F-Stop`'s, the aperture opens and closes to control how much light is allowed into the camera's sensor or film by physically blocking off lights from specific angles, similar to the pupil of an eye.
```

File: content/news/2024-02-03-bevy-0.13/index.md [hunk 3]
```diff
@@ -834,11 +834,34 @@ commands.spawn((
...
+* **Shutter Speed**: How quickly the shutter closes, which controls the duration of time that light is allowed to enter the camera.
```
Comment (TrialDragon) [inline content/news/2024-02-03-bevy-0.13/index.md:844]:
```suggestion
* **Shutter Speed**: How long the camera's shutter is open, which controls the duration of time that the camera's sensor or film is exposed to light.
```

File: content/news/2024-02-03-bevy-0.13/index.md [hunk 4]
```diff
@@ -834,11 +834,34 @@ commands.spawn((
...
+* **ISO Sensitivity**: How sensitive the camera is to light. A higher value indicates a higher sensitivity to light.
```
Comment (TrialDragon) [inline content/news/2024-02-03-bevy-0.13/index.md:845]:
```suggestion
* **ISO Sensitivity**: How sensitive the camera's sensor or film is to light. A higher value indicates a higher sensitivity to light.
```

**Review by TrialDragon — APPROVED** [review]
Non-blocking nit picks on explanations of how camera's exposure and the control of it work; otherwise LGTM.

File: content/news/2024-02-03-bevy-0.13/index.md (hunk 1 above)
Comment (coreh) [reply content/news/2024-02-03-bevy-0.13/index.md:841]: Why the square quotes around “real world” and “exposure settings”? 😄

**Comment by DGriffin91** [comment]
The images here look like before [my 0.13 updates](https://github.com/DGriffin91/bevy_sponza_scene/tree/bevy13) that I think improve the look of the scene a bit. If you want it with those (included 7 since 9.7 looks reasonable):

(7, 9.7, 15)

File: content/news/2024-02-03-bevy-0.13/index.md (hunk 1 above)
Comment (cart) [reply content/news/2024-02-03-bevy-0.13/index.md:841]: I do tend to be overzealous with the double quotes. I like using them to group and introduce concepts. I'll defend "exposure settings" in that context. But "real world" is straight up ambiguous and confusing :laughing:

Comment (cart) [reply content/news/2024-02-03-bevy-0.13/index.md:841]: > Part of the broader confusion here for those unfamiliar with photography is "why do we need multiple"? "Can't we just set a brightness level?"

This is reasonable / I see the value now. I'm in!

Comment (cart) [reply content/news/2024-02-03-bevy-0.13/index.md:841]: I like the added clarity!

>  the amount of light that reaches the camera's sensor or film

However this actually excludes the ISO Sensitivity, which is a process that happens _as_ the light hits the sensor or film

File: content/news/2024-02-03-bevy-0.13/index.md (hunk 2 above)
Comment (cart) [reply content/news/2024-02-03-bevy-0.13/index.md:843]: Agreed. Fixed!

**Comment by cart** [comment]
> The images here look like before [my 0.13 updates](https://github.com/DGriffin91/bevy_sponza_scene/tree/bevy13) that I think improve the look of the scene a bit. If you want it with those (included 7 since 9.7 looks reasonable):

I did actually use that branch. But I think I somehow missed the brightness multiplier (I set new values) and somehow removed SSAO and the environmentmaplight? Really not sure what happened there.

Idk maybe I accidentally re-applied an old changes. Your results are definitely better :)

File: content/news/2024-02-03-bevy-0.13/index.md (hunk 1 above)
Comment (TrialDragon) [reply content/news/2024-02-03-bevy-0.13/index.md]: O, right. I forgot the classic definition of exposure excludes the ISO.  Is the edit I made better?

**Comment by cart** [comment]
One small thing for future reference: our image comparison widget is hard-coded to 16:9 aspect ratio, so these images need some edits.

File: content/news/2024-02-03-bevy-0.13/index.md (hunk 1 above)
Comment (cart) [reply content/news/2024-02-03-bevy-0.13/index.md]: Used essentially your phrasing but tweaked a couple of things.

**Comment by ickk** [comment]
> One small thing for future reference: our image comparison widget is hard-coded to 16:9 aspect ratio, so these images need some edits.

@cart unless something has changed since it was written, you should be able to override the aspect ratio of image comparison widgets case-by-case by setting `style="aspect-ratio: {width} / {height}"` in the markup

**Pull request merged**